package.json

```diff
@@ -38,14 +38,13 @@
   "devDependencies": {
     "@crowdstrike/commitlint": "^8.0.0",
     "eslint": "^9.0.0",
-    "eslint-config-crowdstrike": "10.1.0",
-    "eslint-config-crowdstrike-node": "4.0.0-beta.1",
+    "eslint-config-crowdstrike-node": "4.0.0-beta.2 || ^4.0.0",
     "remark-cli": "^12.0.0",
     "remark-preset-lint-crowdstrike": "^4.0.0",
     "renovate-config-standard": "^2.0.0",
     "standard-version": "^9.0.0"
   },
   "peerDependencies": {
-    "eslint": ">=8.57.0"
```

Comment:
What's wrong with the before?

Reply:
Thought it was a little more explicit but totally ok with it being like before,  i'll revert